### PR TITLE
fix(DB): Adding Mor'ogal's gossip

### PR DESCRIPTION
**Pull request by claudiodfc**
## Changes Proposed:
-  Added a Gossip menu to Mor'ogal.

## Issues Addressed:

- Closes not reported issue.

## SOURCE:

https://wowpedia.fandom.com/wiki/Mor%27rogal

## Tests Performed:

image

## How to Test the Changes:

1. .go c id 11861

## Known Issues and TODO List:

None

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1638446224179942600.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1638446224179942600');
+
+UPDATE `creature_template` SET `gossip_menu_id`='1' WHERE  `entry`=11861;
+
+DELETE FROM `gossip_menu`WHERE `MenuID` = 1 AND `TextID` = 8;
+INSERT INTO `gossip_menu` (`MenuID`, `TextID`) VALUES (1, 8);
+
+DELETE FROM `npc_text`WHERE `ID` = 8;
+INSERT INTO `npc_text` (`text0_0`, `ID`) VALUES ('Stonetalon Mountains has many perilous caves through out this region. Exploring these caves can hold many adventures for those willing to risk life for glory.', 8);
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1638446224179942600.sql:9]: Please keep them in the correct order.

File: data/sql/updates/pending_db_world/rev_1638446224179942600.sql [hunk 2]
```diff
@@ -0,0 +1,9 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1638446224179942600');
+
+UPDATE `creature_template` SET `gossip_menu_id`='1' WHERE  `entry`=11861;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1638446224179942600.sql:3]: dont use quotes around ints

**Review by Nefertumm — APPROVED** [review]

**Comment by Nyeriah** [comment]
I heard gossip_menu is sniffed, needs confirmation from @acidmanifesto

**Comment by claudiodfc** [comment]
You're right, this sniff was already on the database! My bad for not checking.

**Comment by acidmanifesto** [comment]
> I heard gossip_menu is sniffed, needs confirmation from @acidmanifesto 

Gossip is sniffed. Both id, locale, and message.
All you need to do is click it/view it for it to be logged in the sniffer.

**Pull request merged**